internal/oci/cpugroup.go

```diff
@@ -0,0 +1,20 @@
+package oci
+
+import (
+	"context"
+
+	"github.com/Microsoft/hcsshim/internal/uvm"
+)
+
+// HandleCPUGroupSetup will parse the cpugroupID annotation and setup the cpugroup for `vm`
+func HandleCPUGroupSetup(ctx context.Context, vm *uvm.UtilityVM, annotations map[string]string) error {
```

Comment:
Is it just me, or is this function not actually used anywhere?

Reply:
Fixed that :p its hard cherry picking files from other branches lol